plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiSmokeTest.java

```diff
@@ -544,6 +545,32 @@ public void testHudiLongTimestampType()
         testTimestampMicros(HiveTimestampPrecision.NANOSECONDS, LocalDateTime.parse("2020-10-12T16:26:02.906668"));
     }
 
+    @Test
+    public void testHudiCustomKeygenEpochMillisPartitionedTables()
+    {
+        // NOTE: As of now, the partition_path value that is synced to metastore will be returned instead of the raw value that is used by the keygen
+        Session session = SessionBuilder.from(getSession()).build();
+        @Language("SQL") String actualQuery = "SELECT _hoodie_partition_path, part_country, part_date FROM " + HUDI_CUSTOM_KEYGEN_PT_V8_MOR;
```

Comment:
nit: rename for readability (regenerate table too)
```suggestion
        @Language("SQL") String actualQuery = "SELECT _hoodie_partition_path, partition_field_country, partitition_field_date FROM " + HUDI_CUSTOM_KEYGEN_PT_V8_MOR;
```

Reply:
Done!